### PR TITLE
IsNullFixer must be run before YodaStyleFixer

### PR DESCRIPTION
**Pull request by kubawerlos**
@SpacePossum what do you think?

### CODE REVIEW

**Comment by SpacePossum** [comment]
Looking good thanks!
Please add an entry here as well https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.2.13/tests/AutoReview/FixerFactoryTest.php#L59

File: tests/AutoReview/FixerFactoryTest.php [hunk 1]
```diff
@@ -83,6 +83,7 @@ public function provideFixersPriorityCases()
             [$fixers['function_to_constant'], $fixers['no_trailing_whitespace']], // tested also in: function_to_constant,no_trailing_whitespace.test
             [$fixers['function_to_constant'], $fixers['no_whitespace_in_blank_line']], // tested also in: function_to_constant,no_whitespace_in_blank_line.test
             [$fixers['indentation_type'], $fixers['phpdoc_indent']],
+            [$fixers['is_null'], $fixers['yoda_style']],
```
Comment (keradus) [inline tests/AutoReview/FixerFactoryTest.php]: please add comment with a file as well (as when there is lack of comment, we know we don't have itest for given priority test yet)

Comment (kubawerlos) [reply tests/AutoReview/FixerFactoryTest.php]: added and rebased

**Comment by keradus** [comment]
I wonder, as long as this PR itself is good, maybe we shall drop `yoda` part of `IsNullFixer`, as yoda is handled fully by dedicated rule.

**Comment by kubawerlos** [comment]
I thought about it too and created an issue with more details: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3399

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**